src/tokenize/token.ts

```diff
@@ -79,52 +79,52 @@ export const ERRORTOKEN = tokens.ERRORTOKEN;
 export const COMMENT = tokens.COMMENT;
 export const NL = tokens.NL;
 
-export const EXACT_TOKEN_TYPES: { [token: string]: tokens } = {
```

Comment:
so are `Map`'s faster than object access?

Reply:
They seemed to be... Or at least that's what the profiler indicated.

Reply:
maybe because it doesn't have to walk the prototype. I remember you doing something with that in skulpt too.

Reply:
if you want to avoid prototype walks you could try `__proto__: null`